src/browser/views.py

```diff
@@ -343,12 +377,16 @@ def repo_collaborators_add(request, repo_base, repo):
     '''
     username = request.user.get_username()
     collaborator_username = request.POST['collaborator_username']
+    db_privileges = request.POST.getlist('db_privileges')
```

Comment:
That's reasonable to do... but isn't optimization a bit premature here? My impression is that browser is going to get a facelift in the near future.